### DISCUSSION TITLE
How to change nginx port

### DISCUSSION THREAD

**Discussion by greg811** [swizzin/swizzin#1093] [Q&A + Help]
I'm trying to change the nginx port to a different one as I also have Virtualmin installed with an apache server.
I changed the nginx ports from 80 to 81 and from 443 to 444. Nginx seems to be working, but I can't log in to swizzin, it keeps showing the login screen. ruTorrent seems to work fine.
Why can't I log in to swizzin after changing the port? On the standard port 443, login works fine.

**Comment by liaralabs** [comment]
You may have better luck if you disable the forms login
